cpp/test/sg/fil_test.cu

```diff
@@ -463,13 +463,13 @@ class TreeliteFilTest : public BaseFilTest {
       switch (ps.leaf_payload_type) {
         case fil::leaf_value_t::FLOAT_SCALAR:
           // default is fil::FLOAT_SCALAR
-          TL_CPP_CHECK(builder->SetLeafNode(key, output.f));
+          builder->SetLeafNode(key, output.f);
```

Comment:
Why no more TL_CPP_CHECKs for these functions?

Reply:
https://github.com/rapidsai/cuml/pull/2263#discussion_r426039521